content/reference/endpoints/taxes.md

```diff
@@ -17,8 +19,40 @@ id               | Integer | Read       | Tax's id.
 name             | [Object](/reference/enums#formats)   | Read       | Tax's name, list of locales described in [enums section](/reference/enums#locales).
 percentage       | [Decimal](/reference/enums#formats)  | Read       | Tax's percentage.
 -----------------|---------|------------|------------
-created_at       | [Time](/reference/enums#formats) | Read         | Tax's create time.
-updated_at       | [Time](/reference/enums#formats) | Read         | Tax's update time.
+created_at       | [Time](/reference/enums#formats) | Read       | Tax's create time.
+updated_at       | [Time](/reference/enums#formats) | Read       | Tax's update time.
+start_date       | [Date](/reference/enums#formats) | Read       | Tax's start date.
+end_date         | [Date](/reference/enums#formats) | Read       | Tax's end date.
+{: class="table table-bordered"}
+  </div>
+  <div class="tab-pane" id="rates_read" markdown="1">
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | Tax's id.
+-----------------|---------|------------|------------
+name             | [Object](/reference/enums#formats)   | Read       | Tax's name, list of locales described in [enums section](/reference/enums#locales).
+notes            | String  | Read       | Tax's private notes.
+percentage       | [Decimal](/reference/enums#formats)  | Read       | Tax's percentage.
+-----------------|---------|------------|------------
+created_at       | [Time](/reference/enums#formats) | Read       | Tax's create time.
+updated_at       | [Time](/reference/enums#formats) | Read       | Tax's update time.
+start_date       | [Date](/reference/enums#formats) | Read       | Tax's start date.
```

Comment:
@Azdaroth , I think it's a good practice to document all params/attrs, even they're more then clear.